app.py

```diff
@@ -1,23 +1,115 @@
+"""
+Login / Subscribe Page
+- user inputs phone number + email 
+- validations
+    - we should show a confirmation message if the email
+  is indeed a too good to go account, if not  
+
+"""
 import os
 
-from flask import Flask
+from flask import Flask, jsonify, request
+from flask_cors import CORS
 from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
 from tgtg import TgtgClient
+from twilio.rest import Client
 
 from models import Credential, db, Favorite, Subscriber
 
+SECRET_KEY = os.environ.get("SECRET_KEY")
 DATABASE_URI = os.environ.get("DATABASE_URI")
+twilio_account_sid = os.environ.get("twilio_account_sid")
+twilio_auth_token = os.environ.get("twilio_auth_token")
+twilio_phone_number = os.environ.get("twilio_phone_number")
+
+twilio_client = Client(twilio_account_sid, twilio_auth_token)
 
 app = Flask(__name__)
+app.config["SECRET_KEY"] = SECRET_KEY
 app.config['SQLALCHEMY_DATABASE_URI'] = DATABASE_URI
 
 db.init_app(app)
+CORS(app, resources={r"/*": {"origins": "http://localhost:3000"}})
 
 
+@app.route('/check_if_favorites_available')
+def check_if_favorites_available():
+    subscribers = Subscriber.query.all()
+    for subscriber in subscribers:
+        credentials = subscriber.credential
+        if credentials:
+            credential = credentials[0]
+            client = TgtgClient(access_token=credential.access_token, refresh_token=credential.refresh_token, user_id=credential.user_id, cookie=credential.cookie)
+            items = client.get_items()
+            for item in items:
+                item_name = item.get('display_name')
+                item_available = item.get('items_available', 0) > 0
+            
```

Comment:
Sorry, just seeing this. No need to come back to it now, but it would typically work something like:

```
if item is None:
    continue

## do other stuff
```

The `else` wouldn’t be needed because, if the `if` clause is True, it would just skip the rest of the code for that item and move to the next.

Reply:
Gotcha, changed it to this:

for item in items:
                if item is None:
                    continue
                item_name = item.get('display_name', 0)
                item_available = item.get('items_available', 0) > 0
                item_id = int(item.get('item_id', 0))

Reply:
wait, sorry, this version's easier to read:
```bash
for item in items:
if item is None:
continue
item_name = item.get('display_name', 0)
item_available = item.get('items_available', 0) > 0
item_id = int(item.get('item_id', 0))